runtime/lib/ttnn/program.cpp

```diff
@@ -74,6 +78,7 @@ class ProgramExecutor {
     for (const ::tt::target::ttnn::Operation *op : *program->operations()) {
       LOG_DEBUG(LogType::LogRuntimeTTNN,
                 "Executing operation: ", op->debug_info()->c_str());
+      TracyMessage(op->loc_info()->c_str(), op->loc_info()->size());
```

Comment:
Should this be wrapped in an ifdef `#ifdef TT_RUNTIME_ENABLE_PERF_TRACE` as well? If so please create a separate function for this.

Reply:
Yeah maybe create a generic function or if this is the only use-case create a function that has the ifdef in the function, and just does a noop if the ifdef isn't satisfied.

Reply:
yeah it's the only usecase. I'm just going to do:

```cpp
#ifdef TT_RUNTIME_ENABLE_PERF_TRACE
TracyMessage(...);
#endif 
```